front/src/common/ReleaseInformations/LicenseAttributions.tsx

```diff
@@ -11,6 +11,18 @@ type AttributionLicense = {
   license: string;
 };
 
+// Libs which are not in the package.json, so we add it statically
+const COLLABORATIONS: AttributionLicense[] = [
+  {
+    license: 'http://www.apache.org/licenses/LICENSE-2.0',
+    publisher: 'Apache',
+    copyright: '© 2024 Swiss Federal Railways SBB AG',
+    version: '2.9.9',
```

Comment:
Nit: can we leave out the version? It will be cumbersome to keep in sync manually.